### PR TITLE
Fix pytz dependency issue

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
LGTM

**Comment by TimothyAsir** [comment]
@r0h4n , @nthomas-redhat , @shtripat , please review

File: tendrl-commons.spec [hunk 1]
```diff
@@ -25,6 +25,7 @@ Requires: python-urllib3
 Requires: python-six
 Requires: python-docutils
 Requires: python-yaml
+Requires: pytz
```
Comment (r0h4n) [inline tendrl-commons.spec]: are the rpms "namespaces" and "pytz" both available on epel/el7/koji?

Comment (TimothyAsirJeyasing) [inline tendrl-commons.spec]: namespaces rpm available in our fedora copr repo and the pytz is available in centos-base repo

**Pull request merged**